apps/central/src/components/LastUpdateField.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <span v-if="loading"><Spinner></Spinner></span>
+  <span v-else-if="update">
+    <a :href="`/${this.schema}/settings/#/log`">
+  {{ formatStamp(update.stamp) }} ({{update.tableName}})
+  </a>
+  </span>
+  <span v-else><em>failed to fetch update</em></span>
+</template>
+
+<script>
+import { request } from "graphql-request";
+import { Spinner } from "molgenis-components";
+export default {
+  name: "LastUpdateField",
+  components: { Spinner },
+  props: {
+    schema: {
+      String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+        update: null,
+      loading: true
+    };
+  },
+  methods: {
+    formatStamp(stamp) {
+      const date = new Date(stamp);
+      return date.toLocaleDateString();
+    },
+  },
+  async created() {
+    const resp = await request(
+      `/${this.schema}/settings/graphql`,
+      "{_changes(limit: 1) {operation, stamp, userId, tableName}}"
```

Comment:
says 'failed' if no records have yet been added

Reply:
btw, probably should have done a better review on the previous PR, because 'stamp' is not the greates identifier

Reply:
agree i think we can change it to timeStamp in gql without the need to change it in the db

Reply:
> says 'failed' if no records have yet been added

I thinks thats reasonable, could change to  make a difference between , not enabled , empty , failed and succes but just having n/a or data / failed works i think